models/src/main/java/com/vimeo/networking2/enums/ScopeType.kt

```diff
@@ -0,0 +1,20 @@
+package com.vimeo.networking2.enums
+
+/**
+ * All scopes that can be passed to the API. They determine what information you
+ * could get and your permissions in the Vimeo API.
```

Comment:
Might be helpful to point to `https://developer.vimeo.com/api/authentication#scopes` in these docs so devs can go do more reading around "Enables your app to... "